app/components/ui/eventCard.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Event } from '@prisma/client';
+
+export function EventCard({ name, date, description, imgUrl, imgAlt, location }: Event) {
```

Comment:
Currently, we have only very basic UI components in the `/ui/` folder, like `Button` or `Form` that can be reused across our app. Should we maybe move this file into the `modules/` folder, where we sort stuff per feature?

Maybe a `modules/events/` folder with `card.tsx` for the event card?